### ISSUE TITLE
lots of segfaults in new v2 arg processing

### ISSUE THREAD

**Issue by edsantiago** [containers/libpod#6043]
Basically, too many instances of `args[0]` (without checking len(args) to count. See #6041

**Comment by edsantiago** [comment]
Suggestion: pull #6041 into your tree, run `bats test/system/015-help.bats`, fix whatever it complains about, repeat. Some of the fixes seem easy, some (podman container rm) are harder because the code needs to recognize `-l`. This is going to take a while, methinks.

**Comment by edsantiago** [comment]
Here's what I think is the full list:
```
podman container commit
podman container create
podman container exec
podman container export
podman container kill (exit=0)
podman container rm (exit=0)
podman container run
podman container stop (exit=0)
podman container umount (exit=0)
podman container wait (exit=0)
podman  exec
podman image pull
podman  run
```

**Comment by edsantiago** [comment]
Fixed by #6055
